tests/test_wave_01.py

```diff
@@ -139,14 +140,15 @@ def test_moves_movie_from_watchlist_to_watched():
     updated_data = watch_movie(janes_data, movie_to_watch["title"])
 
     # Assert
-    assert len(updated_data["watchlist"]) is 1
-    assert len(updated_data["watched"]) is 2
+    assert len(updated_data["watchlist"]) == 1
+    assert len(updated_data["watched"]) == 2
+    assert HORROR_1 in updated_data["watched"]
```

Comment:
These asserts work great! One way you could make this slightly more robust is by checking that the entire list matches what you expected. This makes sure to catch any bugs where the first element in the list was accidentally overwritten or modified.